### DISCUSSION TITLE
Tables are not populating while trying to use the seed command with comma separated --files

### DISCUSSION THREAD

**Discussion by GayathriMohan** [adonisjs/core#2325] [Help] [answered]
I am trying to seed multiple files at the same time in the local system as well as in the Heroku portal for hosting the application using the command:
`adonis seed --files=Seeder1.js,Seeder2.js,CommonSeeder.js --force`
It prompts that the database has seeded. However, there isn't any data in the 3 tables after running the command.

It gets populated if I run the seed commands separately as:
`adonis seed --files=Seeder1.js`
`adonis seed --files=Seeder2.js`
`adonis seed --files=CommonSeeder.js`

Is there a workaround for seeding the multiple files in one command?

**Comment by thetutlage** [comment, marked as answer]
Duplicate of https://github.com/adonisjs/core/discussions/2329
